molo/requirements.txt

```diff
@@ -0,0 +1,3 @@
+rapidpro-python==2.6.1
+NewlineJSON
+google-cloud-bigquery
```

Comment:
Why do we need to install the bigquery and rapidpro libraries? what are we using them for?